### ISSUE TITLE
Windows 11 desktop scaling issue

### ISSUE THREAD

**Issue by Andrewien** [debauchee/barrier#1894]
### What happened?

Mouse gets stuck on an invisible line whose location changes based on scaling, at normal scale it has no issue. This issue was resolved when i uninstalled barrier.

### Version

v2.3.4

### If applicable, where did you install Barrier from?

I got it from it's site.

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

Windows 11, but it wasn't active, this bug was in effect even while barrier was offline.

### Any other information

This bug occurs even if barrier isn't running.

**Comment by Eduardocoutto** [comment]
It happens to me too. But when I set the system scale to 100% everything is correct.

**Comment by Andrewien** [comment]
Yes, but my screen is small and I can't see clearly if it isn't set higher.

**Comment by timriker** [comment]
See #1713

Windows 2.3.4 works fine as server for me.

**Comment by Andrewien** [comment]
Is my explanation unclear ? What you redirected me to isn't the same issue, mainly that this issue occurs by somehow just having barrier installed. I wasn't running it, and uninstalling barrier fixed it. Describing the issue itself is harder because it's unusual, my mouse got stuck on an invisible edge onscreen on the same computer, not across desktop screens all while barrier wasn't active and was installed. My mouse was able to pass this line if I forced my mouse across my mouse pad quicker and it happened both while trying to move the mouse up and down.

**Comment by erothoff** [comment]
I found a better solution for the scaling issue of  #1713 and this  #1894.  Yes the problem is the scaling, but instead of scaling your 4K monitor to get the larger screen, you can change the resolution down to 2560x1440. It is the same size as having the scaling at 150% but it also allows Barrier 2.4.0 to work.  I would rather the scaling issue be fixed, but at least there is another workaround.

**Comment by timriker** [comment]
> Is my explanation unclear ? What you redirected me to isn't the same issue, mainly that this issue occurs by somehow just having barrier installed. I wasn't running it, and uninstalling barrier fixed it. Describing the issue itself is harder because it's unusual, my mouse got stuck on an invisible edge onscreen on the same computer, not across desktop screens all while barrier wasn't active and was installed. My mouse was able to pass this line if I forced my mouse across my mouse pad quicker and it happened both while trying to move the mouse up and down.

I had similar issues with the newer release, but it looks like you were already on 2.3.4 so I'm not sure what's causing your issue. This is probably not a direct duplicate of #1713  but it sounds like it's still related to resolution scaling on windows.

**Comment by RoyeA** [comment]
Same problem here.
Used the workaround of changing the resolution until the issue will be resolved.

**Comment by igordovgaluk** [comment]
the problem has a fix here https://github.com/debauchee/barrier/pull/1506#issuecomment-1013758495

**Comment by RoyeA** [comment]
1506 doesn't resolve the issue. the issue exists in the latest release.

**Comment by igordovgaluk** [comment]
and #1506 has been merged after v2.4.0 the latest at the moment was released

**Comment by RoyeA** [comment]
> and #1506 has been merged after v2.4.0 the latest at the moment was released

So, I'll reinstall and try it ( It should have a different number ).

**Comment by Taknok** [comment]
See this comment as a workaround : https://github.com/debauchee/barrier/issues/1638#issuecomment-1094301110
